src/components/DisplayEmbedTag.jsx

```diff
@@ -45,37 +22,42 @@ export default class DisplayEmbedTag extends React.Component {
   }
 
   componentWillReceiveProps(nextProps) {
-    if (nextProps.embedTag !== this.props.embedTag) {
+    if (nextProps.embedTag.id !== this.props.embedTag.id) {
       this.handleFetchImage(nextProps.embedTag);
     }
   }
 
   handleFetchImage(embedTag) {
-    const embed = parseEmbedTag(embedTag);
-    if (!embed || embed.resource !== 'image') {
+    if (!embedTag || embedTag.resource !== 'image') {
       return;
     }
 
-    fetchImage(embed.id).then(image => {
```

Comment:
Med mindre det skal vises frem noe data fra som kommer fra APIet er det ikke lengre behov for å fetche fra APIet. Det går an å hente ut et raw bilde direkte med en id slik: https://test.api.ndla.no/image-api/raw/id/1 

URLen som ligger på bilde-embed-tags (data-url) peker fremdeles på endepunktet for å fetche meta-data om bildet. Dersom det er slik at meta-data for bilder aldri blir fetchet kan sikkert `data-url` endres til å peke direkte på raw-endepunktet, men jeg regner med at det brekker en del eksisterende funksjonalitet.